spec/client_spec.rb

```diff
@@ -74,8 +71,12 @@
   end
 
   context 'post' do
-    it 'returns hash on success' do
-      expect(client.post_posts).to be_an(Hash)
+    it 'returns Takeout::Response on success' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.